src/lsm/lsm_meta.c

```diff
@@ -349,6 +349,13 @@ __lsm_meta_read_v1(
 	 * Ignore any other values: the metadata entry might have been
 	 * created by a future release, with unknown options.
 	 */
+	/*
+	 * If the default merge_min was not overridden, calculate it now.  We
+	 * do this here so that trees created before merge_min was added get a
```

Comment:
The comment and code is copied directly from `lsm_meta_read_v0`.  It may be worth refactoring some of the code so we have that code once.